sql/core/src/main/scala/org/apache/spark/sql/execution/DataSourceScanExec.scala

```diff
@@ -102,7 +102,7 @@ case class RowDataSourceScanExec(
     filters: Set[Filter],
     handledFilters: Set[Filter],
     rdd: RDD[InternalRow],
-    @transient relation: BaseRelation,
+    @transient override val relation: BaseRelation,
```

Comment:
do we need `override`?

Reply:
Oh, I think I forgot to update here. Per previous discussion, we should replace base definition to 'def' instead of adding these `override`s.